### PR TITLE
Bring HTML inputs into view automatically

### PR DESCRIPTION
**Pull request by blasten**
Fixes https://github.com/flutter/flutter/issues/75667

The inputs are hidden because `InputAwareWebView` manages focus within the webview. This extension was added to workaround edge cases with virtual displays, but those are solved in the hybrid composition implementation.

WebView doesn't have Junit tests, unfortunately.  I think we should clean up webview, and add platform tests similar to how we did in ads.

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[C++, Objective-C, Java style guides]: https://github.com/flutter/engine/blob/master/CONTRIBUTING.md#style
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat
[pub versioning philosophy]: https://dart.dev/tools/pub/versioning

### CODE REVIEW

**Review by bparrishMines — APPROVED** [review]
LGTM

File: packages/webview_flutter/lib/webview_flutter.dart [hunk 1]
```diff
@@ -109,6 +109,7 @@ class SurfaceAndroidWebView extends AndroidWebView {
           layoutDirection: TextDirection.rtl,
           creationParams: MethodChannelWebViewPlatform.creationParamsToMap(
             creationParams,
+            usesHybridComposition: true,
```
Comment (stuartmorgan) [inline packages/webview_flutter/lib/webview_flutter.dart:112]: We should test that creating a `SurfaceAndroidWebView` sets this in the method channel to ensure the Dart side of this stays working.

**Review by stuartmorgan — COMMENTED** [review]
> WebView doesn't have Junit tests, unfortunately. I think we should clean up webview, and add platform tests similar to how we did in ads.

How hard is wiring up unit tests in Java? Obviously backfill is a major project, but can we add unit tests of just this piece now, so we're heading in the right direction?

File: packages/webview_flutter/lib/webview_flutter.dart (hunk 1 above)
Comment (blasten) [reply packages/webview_flutter/lib/webview_flutter.dart:112]: I ended up adding an e2e test. :)

Comment (stuartmorgan) [reply packages/webview_flutter/lib/webview_flutter.dart:112]: We still really need a unit test harness for this package, but I'll take it :)

**Review by stuartmorgan — APPROVED** [review]
LGTM

**Comment by blasten** [comment]
The failure in the CI doesn't make sense. The test is skipped, but somehow it appears to be running on iOS.  I'm not sure how this is possible.
